### PR TITLE
Additional Logic for Disposal

### LINKED ISSUE #5732: The disposal chute in waste disposal goes back to waste disposal

### PR DESCRIPTION
**Pull request by WhiteHusky**
disposalholders and Delivery Chutes got a change along with the old wrapsortjunction.

This fixes #5732 by giving a additional variable to adjust for Delivery Chutes as defaultTag which means the chute in Disposals has been given the defaultTag of 2 so it returns recycled items to the delivery outlet in cargo. Additionally in Disposals, glass was placed to redirect items flying out of the outlet and make sure it lands in the recycler. Oh, and chutes don't tag packages if they have not been, this will be explained.

disposalholders check if destination tags inside are different, causing them to send back to cargo (tagged 0) with the variable mixed set. Wraped mail is also checked for, setting the variable mail.

The old wrapsortjunction was updated and added as part of cargo bay, behind the existing sorting junction, with variables sortMail and sortMixed to control if it should sort mail or mixed items. This will send mixed items and mail to the delivery outlet in cargo to be corrected or mailed correctly.

Changed the variable has_fat_guy to tooBig since it's more appropriate to consider if the disposalholder is too big and get stuck as a result

In all, this PR ensures tagged items don't get sent incorrectly because of a differently tagged set of items, recycled items can go right to the Cargo Bay through the chute, and untagged items are sent back to Cargo Bay

### CODE REVIEW

**Comment by CrAzYPiLoT-SS13** [comment]
>hasFatGuy
Thank god you renamed this

File: code/modules/recycling/disposal.dm [hunk 1]
```diff
@@ -499,10 +502,11 @@
 	var/active = 0	// true if the holder is moving, otherwise inactive
 	dir = 0
 	var/count = 1000	//*** can travel 1000 steps before going inactive (in case of loops)
-	var/has_fat_guy = 0	// true if contains a fat person
+	var/tooBig = 0	// true if the holder is too big and can get stuck
```
Comment (Fox-McCloud) [inline code/modules/recycling/disposal.dm]: Don't unnecessarily change var names, unless there's a reason.

**Comment by Fox-McCloud** [comment]
This has a lot of implied `src`'s. Please remove them.

File: code/modules/recycling/disposal.dm (hunk 1 above)
Comment (WhiteHusky) [inline code/modules/recycling/disposal.dm]: Understood. Going back to revert them.

**Comment by WhiteHusky** [comment]
@Fox-McCloud I've noticed it too and in retrospect they don't really need to exist. Was following the original code assuming there was a valid reason.

**Comment by WhiteHusky** [comment]
Canceling PR. Going to make a new PR of just the tweak fixing #5732.

**Pull request closed without merging**